gips/data/prism/prism.py

```diff
@@ -35,14 +35,15 @@
 
 from gips.data.core import Repository, Asset, Data
 
-from gips.utils import settings, VerboseOut, List2File
+from gips.utils import settings, List2File
+from gips import utils
 
 from gippy import GeoImage
 
 from numpy import mean
 
 __author__ = "Ian Cooke <icooke@ags.io>"
-__version__ = '0.0.1'
+__version__ = '0.1.0'
```

Comment:
I can't find any use of this value; since it was already there it's harmless to leave it in though.